yellow-paper/docs/public-vm/gen/_InstructionSet.mdx

```diff
@@ -498,7 +498,26 @@ Subtraction (a - b)
 
 [![](./images/bit-formats/SUB.png)](./images/bit-formats/SUB.png)
 
-### <a id='isa-section-div'/>`DIV` (0x02)
+### <a id='isa-section-mul'/>`MUL` (0x02)
+Subtraction (a - b)
+
+[See in table.](#isa-table-mul)
+
+- **Category**: arithmetic
+- **Flags**: 
+	- **indirect**: Toggles whether each memory-offset argument is an indirect offset. 0th bit corresponds to 0th offset arg, etc. Indirect offsets result in memory accesses like `M[M[offset]]` instead of the more standard `M[offset]`.
+	- **in-tag**: The [tag/size](./state-model#tags-and-tagged-memory) to check inputs against and tag the destination with.
+- **Args**: 
+	- **aOffset**: memory offset of the operation's left input
+	- **bOffset**: memory offset of the operation's right input
+	- **dstOffset**: memory offset specifying where to store operation's result
+- **Expression**: `M[dstOffset] = M[aOffset] * M[bOffset] mod 2^k`
+- **Tag checks**: `T[aOffset] == T[bOffset] == in-tag`
+- **Tag updates**: `T[dstOffset] = in-tag`
+- **Bit-size**: 128
+
+
+### <a id='isa-section-div'/>`DIV` (0x03)
```

Comment:
@jeanmon since I know you've been giving some thought to the `MUL` instruction, I thought I'd ping you to take a look at @Maddiaa0's spec here.

Reply:
Ive just notice i didnt properly update all the bits from where i copied it from subtraction - will update